compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/stubs/IntrinsicStubs.java

```diff
@@ -55,6 +56,7 @@
 @GeneratedStubsHolder(targetVM = "hotspot", sources = {
                 ArrayIndexOfNode.class,
                 ArrayEqualsNode.class,
+                ArrayFillNode.class,
```

Comment:
Final request, I hope: To enable support for this node in native images, please also make the corresponding change in `SVMIntrinsicStubs` and add an entry in `AArch64StubForeignCallsFeature` (both in the `substratevm` suite).
You can test this by building libgraal, i.e., the GraalVM compiler compiled to a native shared library. For that, run `mx --env libgraal build` in the `vm` suite. The build should fail with the current state of your code, but pass again when you make the requested changes.

Reply:
Sure thing. Done.